<!DOCTYPE html>
<html><head><title>RemoteException</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">3 instantiations of RemoteException</div><div class="rA">System.Management.Automation (3)</div><div class="rG" id="System.Management.Automation"><div class="rF"><div class="rN">engine\ErrorPackage.cs (1)</div>
<a href="../engine/ErrorPackage.cs.html#1342"><b>1342</b>RemoteException re = new <i>RemoteException</i>((!string.IsNullOrWhiteSpace(exceptionMessage)) ? exceptionMessage : errorCategory_Message, serializedException, invocationInfo);</a>
</div>
<div class="rF"><div class="rN">engine\NativeCommandProcessor.cs (1)</div>
<a href="../engine/NativeCommandProcessor.cs.html#1884"><b>1884</b>obj = new ErrorRecord(new <i>RemoteException</i>(errorMessage),</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\remoterunspace.cs (1)</div>
<a href="../engine/remoting/client/remoterunspace.cs.html#2624"><b>2624</b>exception = new <i>RemoteException</i>(</a>
</div>
</div>
<div class="rH">25 references to RemoteException</div><div class="rA">Microsoft.PowerShell.Commands.Utility (2)</div><div class="rG" id="Microsoft.PowerShell.Commands.Utility"><div class="rF"><div class="rN">commands\utility\ImplicitRemotingCommands.cs (2)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs.html#732"><b>732</b><i>RemoteException</i> remoteException = runtimeException as <i>RemoteException</i>;</a>
</div>
</div>
<div class="rA">Microsoft.PowerShell.ConsoleHost (3)</div><div class="rG" id="Microsoft.PowerShell.ConsoleHost"><div class="rF"><div class="rN">host\msh\ConsoleHost.cs (3)</div>
<a href="../../Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs.html#2590"><b>2590</b>e is <i>RemoteException</i> ||</a>
<a href="../../Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs.html#2787"><b>2787</b><i>RemoteException</i> remoteException = e as <i>RemoteException</i>;</a>
</div>
</div>
<div class="rA">System.Management.Automation (20)</div><div class="rG" id="System.Management.Automation"><div class="rF"><div class="rN">engine\ErrorPackage.cs (1)</div>
<a href="../engine/ErrorPackage.cs.html#1342"><b>1342</b><i>RemoteException</i> re = new RemoteException((!string.IsNullOrWhiteSpace(exceptionMessage)) ? exceptionMessage : errorCategory_Message, serializedException, invocationInfo);</a>
</div>
<div class="rF"><div class="rN">engine\Modules\RemoteDiscoveryHelper.cs (2)</div>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#362"><b>362</b><i>RemoteException</i> remoteException = innerException as <i>RemoteException</i>;</a>
</div>
<div class="rF"><div class="rN">engine\NativeCommandProcessor.cs (1)</div>
<a href="../engine/NativeCommandProcessor.cs.html#2374"><b>2374</b>/// Initializes a new instance of the &lt;see cref=&quot;<i>RemoteException</i>&quot;/&gt;</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\Job.cs (3)</div>
<a href="../engine/remoting/client/Job.cs.html#761"><b>761</b>if (!(runtimeException is <i>RemoteException</i> remoteException))</a>
<a href="../engine/remoting/client/Job.cs.html#3396"><b>3396</b><i>RemoteException</i> rException = failureException as <i>RemoteException</i>;</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\remoterunspace.cs (9)</div>
<a href="../engine/remoting/client/remoterunspace.cs.html#1934"><b>1934</b><i>RemoteException</i> re = e as <i>RemoteException</i>;</a>
<a href="../engine/remoting/client/remoterunspace.cs.html#1949"><b>1949</b>(re.ErrorRecord.CategoryInfo.Reason == nameof(<i>RemoteException</i>)))</a>
<a href="../engine/remoting/client/remoterunspace.cs.html#1958"><b>1958</b>if ((e is PSRemotingTransportException) || (e is <i>RemoteException</i>))</a>
<a href="../engine/remoting/client/remoterunspace.cs.html#2990"><b>2990</b>catch (<i>RemoteException</i> e)</a>
<a href="../engine/remoting/client/remoterunspace.cs.html#3050"><b>3050</b>catch (<i>RemoteException</i> e)</a>
<a href="../engine/remoting/client/remoterunspace.cs.html#3111"><b>3111</b>catch (<i>RemoteException</i> e)</a>
<a href="../engine/remoting/client/remoterunspace.cs.html#3156"><b>3156</b>catch (<i>RemoteException</i> e)</a>
<a href="../engine/remoting/client/remoterunspace.cs.html#3216"><b>3216</b>catch (<i>RemoteException</i> e)</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\ConnectPSSession.cs (1)</div>
<a href="../engine/remoting/commands/ConnectPSSession.cs.html#410"><b>410</b>catch (<i>RemoteException</i>)</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\DisconnectPSSession.cs (1)</div>
<a href="../engine/remoting/commands/DisconnectPSSession.cs.html#269"><b>269</b>catch (<i>RemoteException</i>)</a>
</div>
<div class="rF"><div class="rN">engine\remoting\server\ServerRemoteHost.cs (2)</div>
<a href="../engine/remoting/server/ServerRemoteHost.cs.html#427"><b>427</b>catch (<i>RemoteException</i>) { }</a>
<a href="../engine/remoting/server/ServerRemoteHost.cs.html#453"><b>453</b>catch (<i>RemoteException</i>) { }</a>
</div>
</div>
</body></html>